src/EVM/SymExec.hs

```diff
@@ -704,24 +735,87 @@ verify solvers opts preState maybepost = do
           pure (res, leaf)
         let cexs = filter (\(res, _) -> not . isQed $ res) results
         when conf.debug $ putStrLn $ "   Found " <> show (length cexs) <> " potential counterexample(s) in call " <> call
-        pure $ if Prelude.null cexs then (expr, [Qed]) else (expr, fmap toVRes cexs)
+        pure $ (expr, cexs) --]]+ preState expr cexs
   where
-    getCallPrefix :: Expr Buf -> String
-    getCallPrefix (WriteByte (Lit 0) (LitByte a) (WriteByte (Lit 1) (LitByte b) (WriteByte (Lit 2) (LitByte c) (WriteByte (Lit 3) (LitByte d) _)))) = mconcat $ map (printf "%02x") [a,b,c,d]
-    getCallPrefix _ = "unknown"
-    toProps leaf constr post = PNeg (post preState leaf) : constr <> extractProps leaf
-    toPropsFinal leaf constr post = if opts.simp then Expr.simplifyProps $ toProps leaf constr post
-                                                 else toProps leaf constr post
     canBeSat (a, _) = case a of
         [PBool False] -> False
         _ -> True
+
+verifyResults :: VM Symbolic RealWorld -> Expr End -> [(SMTResult, Expr End)] -> (Expr End, [VerifyResult])
+verifyResults preState expr cexs = if Prelude.null cexs then (expr, [Qed]) else (expr, fmap toVRes cexs)
+    where
```

Comment:
We usually put the `where` two spaces indented, and then the `toVRes` can stay where it's at, i.e. 4 spaces indented.

Reply:
Fixed